GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/random.cpp

```diff
@@ -234,19 +234,19 @@ int Random2Class::operator() (int minval, int maxval)
 **	and thus prevents the algorithm from breaking down as a result.
 */
 int Random3Class::Mix1[20] = {
-	static_cast<int>(0xbaa96887), static_cast<int>(0x1e17d32c), static_cast<int>(0x03bcdc3c), static_cast<int>(0x0f33d1b2),
-	static_cast<int>(0x76a6491d), static_cast<int>(0xc570d85d), static_cast<int>(0xe382b1e3), static_cast<int>(0x78db4362),
-	static_cast<int>(0x7439a9d4), static_cast<int>(0x9cea8ac5), static_cast<int>(0x89537c5c), static_cast<int>(0x2588f55d),
-	static_cast<int>(0x415b5e1d), static_cast<int>(0x216e3d95), static_cast<int>(0x85c662e7), static_cast<int>(0x5e8ab368),
-	static_cast<int>(0x3ea5cc8c), static_cast<int>(0xd26a0f74), static_cast<int>(0xf3a9222b), static_cast<int>(0x48aad7e4)
+	(int)0xbaa96887, (int)0x1e17d32c, (int)0x03bcdc3c, (int)0x0f33d1b2,
```

Comment:
The second commit should be merged into the first commit, because they touch the exact same code.